views/index.erb

```diff
@@ -1,8 +1,31 @@
 <h1><%= @title %></h1>
 
 Output generated: <%= Time.now %>
+
 <pre>
 $ <%= @command %>
-
-<%= @output %>
 </pre>
+
+<table class="table table-bordered">
+  <tr>
+    <th>USER</th>
+    <th>PID</th>
+    <th>%CPU</th>
+    <th>%MEM</th>
+    <th>VSZ</th>
+    <th>RSS</th>
+    <th>TTY</th>
+    <th>STAT</th>
+    <th>START</th>
+    <th>TIME</th>
+    <th>COMMAND</th>
+  </tr>
+  <% @output.each do |row| %>
+  <tr>
+    <% row.each do |value| %>
+      <td><%= value %></td>
+    <% end %>
+  </tr>
```

Comment:
Also, if you know for sure that the first columns will have no spaces in the values, just around, you could change the regex to this:

```ruby
def parse_ps(ps_string)
  ps_string.scan(/(([^\s]+\s+){10})(.*)/).map { |p| AppProcess.new(*(p.first.split), p.last) }
end
```

This works because the there are 3 matches, the first is all the columns before the last, the second is the last match of those (to ignore), and the third is the last column i.e.:

```
["efranz    36395  3.6  0.5 374740 89056 ?        Sl   16:50   0:02 ", "0:02 ", "Passenger RackApp: /var/www/ood/apps/sys/dashboard                 "]
```